query_frontend/src/parser.rs

```diff
@@ -791,6 +791,47 @@ fn check_column_expr_validity_in_hash(column: &Ident, columns: &[ColumnDef]) ->
     valid_column.is_some()
 }
 
+/// Builds and checks for the existence of a timestamp key constraint named
+/// __ts_key within the given list of table constraints. If such a constraint
+/// does not exist, the function searches for a unique timestamp column and
+/// creates a new constraint for it.
+fn build_and_check_timestamp_key_constraint(
```

Comment:
I suggest we name it `build_or_infer_timestamp_key_constraint`.

Reply:
Thank you for the suggestion. The name `build_or_infer_timestamp_key_constraint` does seem clearer than the previous one. I'll make the change shortly.

Reply:
See c59aed1a612c608df8e7bcec5b95c06980f69079.